build/make-manifests.sh

```diff
@@ -45,8 +40,33 @@ ls ${BUNDLE_DIR_VERSION}
 # needed to make operator-courier happy
 cp "${BUNDLE_DIR}/node-maintenance-operator.package.yaml" "${BUNDLE_DIR_VERSION}/node-maintenance-operator.package.yaml"
 
-if [ "${HAVE_COURIER}" == "1" ]; then
-    operator-courier verify ${BUNDLE_DIR_VERSION} && echo "OLM verify passed" || echo "OLM verify failed"
+set +e
+
+echo "OLM verify bundle"
+$HOME/.local/bin/operator-courier verify ${BUNDLE_DIR_VERSION}
```

Comment:
This is bad for two main reasons...
- We should not be polluting $HOME if a suitable version already exists
- Downstream builds cannot access the internet

Reply:
make manifest is not run during the travis build.

Reply:
one of the changes here was to add the dockerized courier check, so that we dont't depend on what is installed in the current environment. (see test-courier target in Makefile)